extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/TraceMultiPropagator.java

```diff
@@ -49,7 +49,7 @@
  *   .extract(context, carrier, carrierGetter);
  * }</pre>
  *
- * This class is deprecated and will be removed in 0.14.0. Users should migrate to the
+ * This class is @deprecated and will be removed in 0.14.0. Users should migrate to the
  * MultiTextMapPropagator in the api.
  */
```

Comment:
```suggestion
 * @deprecated Migrate to the MultiTextPropagator in the API.
 */
```

Reply:
Yeah sorry for lack of clarity. This format will render a bit better in IntelliJ I think